### ISSUE TITLE
 No internet connection!

### ISSUE THREAD

**Issue by AMJ-7** [mullvad/dns-blocklists#74]
Hi, I tried to make a new Configuration Profiles for iOS but I can't open any website after connected to: 

server IP: `100.64.0.31`

server URL: `https://adblock.doh.mullvad.net/dns-query`

what's wrong?

**Comment by jbjorkang** [comment]
You are mixing them up.

100.64.0.31 is for the DNS block lists when you're using the VPN. Meaning you select this when using custom DNS in the app.

The URL is for DNS over HTTPS. 

These are separate and cannot be mixed.

Furthermore, DNS over HTTPS does not include all these lists currently. So for the 100.64.0.31 list you've chosen, you need to use the Mullvad VPN app, or a configuration file for OpenVPN or WireGuard.

Encrypted DNS will include all lists in due time.

TL;DR Don't specify the IP with that URL, or use the IP as custom DNS in the app.
